main/modes/games/soko/sokoHelp.c

```diff
@@ -0,0 +1,183 @@
+//==============================================================================
+// Includes
+//==============================================================================
+
+#include "hdw-tft.h"
+#include "sokoHelp.h"
+
+//==============================================================================
+// Defines
+//==============================================================================
+
+#define TEXT_MARGIN_L 18
+#define TEXT_MARGIN_R 13
+
+#define ARROW_BLINK_PERIOD 1000000
+
+//==============================================================================
+// Structs
+//==============================================================================
+
+typedef struct
+{
+    const char* title;
+    const char* text;
+} sokoHelpPage_t;
+
+//==============================================================================
+// Const data
+//==============================================================================
+
+static const char controlTitle[] = "Controls";
+static const char boxTitle[]     = "Boxes";
+
+static const sokoHelpPage_t helpPages[] = {
```

Comment:
@hunterdyar all I want you to review here is this copy. I figured some instructions were better than none. I worked with what I got, theme-wise.